kitchen/bin/run.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
This script is meant to be used for starting kitchen when using the shell scheduler or a local mesos agent. In those scenarios, I wouldn't want it to automatically build. However, I added an error message suggesting that it be built if the uberjar is not found. What do you think?